### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by suntrix**

### CODE REVIEW

File: firebase-auth/src/iosMain/kotlin/dev/gitlive/firebase/auth/user.kt [hunk 1]
```diff
@@ -45,7 +45,7 @@ actual class FirebaseUser internal constructor(val ios: FIRUser) {
         ios.awaitResult<FIRUser, FIRAuthDataResult?> { reauthenticateWithCredential(credential.ios, it) }.run { Unit }
 
     actual suspend fun reauthenticateAndRetrieveData(credential: AuthCredential): AuthResult =
-        AuthResult(ios.awaitResult { reauthenticateAndRetrieveDataWithCredential(credential.ios, it) })
+        AuthResult(ios.awaitResult { reauthenticateAndRetrieveData(credential) })
```
Comment (nbransby) [inline firebase-auth/src/iosMain/kotlin/dev/gitlive/firebase/auth/user.kt:48]: this looks like its calling itself - do you need to revert this back to using https://firebase.google.com/docs/reference/ios/firebaseauth/api/reference/Classes/FIRUser#-reauthenticatewithcredential:completion:

**Review by nbransby — COMMENTED** [review]
Hey @suntrix this is great thanks! Just one comment...

File: firebase-auth/src/iosMain/kotlin/dev/gitlive/firebase/auth/user.kt (hunk 1 above)
Comment (suntrix) [reply firebase-auth/src/iosMain/kotlin/dev/gitlive/firebase/auth/user.kt]: Right, totally missed that. Fixed :)

**Review by nbransby — APPROVED** [review]

**Comment by suntrix** [comment]
@nbransby Any chance to have a new release this these changes this week?

**Comment by nbransby** [comment]
Yes that should be possible

**Comment by suntrix** [comment]
Thank you!

**Comment by suntrix** [comment]
@nbransby I see v1.1.0 got published, but can't see it in https://search.maven.org/search?q=g:dev.gitlive%20AND%20a:firebase-auth

**Comment by nbransby** [comment]
yes there's an issue with the CI which we are looking into

**Comment by Reedyuk** [comment]
@suntrix - it has been published and pushed into maven.org but give it a few hours before checking as they have a level of caching.

**Pull request merged**